packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -206,6 +189,54 @@ class ChromeLauncher {
     ), skipCheck);
   }
 
+  // This is a JSON file which contains configuration from the browser session,
+  // such as window position. It is located under the Chrome data-dir folder.
+  String get _preferencesPath => _fileSystem.path.join('Default', 'preferences');
+
+  // The directory that Chrome uses to store local storage information for web apps.
+  String get _localStoragePath => _fileSystem.path.join('Default', 'Local Storage');
+
+  /// Copy Chrome user information from a Chrome session into a per-project
+  /// cache.
+  void _cacheUserSessionInformation(Directory userDataDir, Directory cacheDir) {
+    final File targetPreferencesFile = _fileSystem.file(_fileSystem.path.join(cacheDir?.path ?? '', _preferencesPath));
+    final File sourcePreferencesFile = _fileSystem.file(_fileSystem.path.join(userDataDir.path, _preferencesPath));
+    final Directory targetLocalStorageDir = _fileSystem.directory(_fileSystem.path.join(cacheDir?.path ?? '', _localStoragePath));
+    final Directory sourceLocalStorageDir = _fileSystem.directory(_fileSystem.path.join(userDataDir.path, _localStoragePath));
+
+    if (sourcePreferencesFile.existsSync()) {
+      targetPreferencesFile.parent.createSync(recursive: true);
+      // If the file contains a crash string, remove it to hide the popup on next run.
+      final String contents = sourcePreferencesFile.readAsStringSync();
+      targetPreferencesFile.writeAsStringSync(contents
```

Comment:
Is there a link you can add in a comment either describing the schema of this file, or at least of this field?

Also, is there a test for the find/replace?

Reply:
I don't believe there's a schema. The existing text (this portion of the PR is existing code) does contain a check for the find / replace for the crash string.

Reply:
I found one of file locations by reading chrome source, we could link to that?

Reply:
Can you link it here, and I'll add to the PR?

Reply:
I can't find it, It might have just been grepping the user data dir for strings

Reply:
I found some info here: https://www.chromium.org/developers/design-documents/preferences; will in-line into the source code.